modules/osdk-csv-manual-annotations.adoc

```diff
@@ -5,75 +5,4 @@
-
-- `disconnected`: Operator supports being mirrored into disconnected catalogs, including all dependencies, and does not require internet access. All related images required for mirroring are listed by the Operator.
-- `cnf`: Operator provides a Cloud-native Network Functions (CNF) Kubernetes plugin.
-- `cni`: Operator provides a Container Network Interface (CNI) Kubernetes plugin.
-- `csi`: Operator provides a Container Storage Interface (CSI) Kubernetes plugin.
-- `fips`: Operator accepts the FIPS mode of the underlying platform and works on nodes that are booted into FIPS mode.
-
-[IMPORTANT]
-====
-When running {op-system-base-full} or {op-system-first} booted in FIPS mode, {product-title} core components use the {op-system-base} cryptographic libraries that have been submitted to NIST for FIPS 140-2/140-3 Validation on only the x86_64, ppc64le, and s390x architectures.
-====
-- `proxy-aware`: Operator supports running on a cluster behind a proxy. Operator accepts the standard proxy environment variables  `HTTP_PROXY` and `HTTPS_PROXY`, which Operator Lifecycle Manager (OLM) provides to the Operator automatically when the cluster is configured to use a proxy. Required environment variables are passed down to Operands for managed workloads.
-
-|`operators.openshift.io/valid-subscription`
-|Free-form array for listing any specific subscriptions that are required to use the Operator. For example, `'["3Scale Commercial License", "Red Hat Managed Integration"]'`.
-
-|`operators.operatorframework.io/internal-objects`
-|Hides CRDs in the UI that are not meant for user manipulation.
-
-|===
-
-[discrete]
-[id="osdk-csv-manual-annotations-examples_{context}"]
-== Example use cases
-
-.Operator supports disconnected and proxy-aware
-[source,terminal]
-----
-operators.openshift.io/infrastructure-features: '["disconnected", "proxy-aware"]'
-----
-
-.Operator requires an {product-title} license
-[source,terminal]
-----
-operators.openshift.io/valid-subscription: '["OpenShift Container Platform"]'
-----
-
-.Operator requires a 3scale license
-[source,terminal]
-----
-operators.openshift.io/valid-subscription: '["3Scale Commercial License", "Red Hat Managed Integration"]'
-----
-
-.Operator supports disconnected and proxy-aware, and requires an {product-title} license
-[source,terminal]
-----
-operators.openshift.io/infrastructure-features: '["disconnected", "proxy-aware"]'
-operators.openshift.io/valid-subscription: '["OpenShift Container Platform"]'
-----
+Operator developers can set certain annotations in the metadata of a cluster service version (CSV) to enable features or highlight capabilities in user interfaces (UIs), such as OperatorHub or the link:https://catalog.redhat.com/software/search?deployed_as=Operator[Red Hat Ecosystem Catalog]. Operator metadata annotations are manually defined by setting the `metadata.annotations` field.
```

Comment:
Do we want to mention any specific file in which this `metadata.annotations` field is present?
Operator metadata annotations are manually defined by setting the `metadata.annotations` field in the ........ file.

Reply:
FWIW, this "Operator metadata annotations" section was previously a subsection of the "Manually-defined CSV fields", but in this PR I've brought it up a level in the TOC. So I suppose it has lost a little of the connective tissue in that way (e.g., that earlier section says "Operator authors must directly modify their cluster service version (CSV) YAML file..."), though I feel that the TOC still works.

The first sentence of the updated content does still mention the file that the annotations are set in ("in the metadata of a cluster service version (CSV)"), but I could still at "in the CSV YAML file" in the suggested spot, too.